.pre-commit-hooks.yaml

```
@@ -0,0 +1,8 @@
+- id: darker
+  name: darker
+  description: "Black reformatting to Python files in only regions changed since last commit"
```

Just a little nitpick:

```suggestion
  description: "Black reformatting to Python files only in regions changed since last commit"
```